src/sentry/templates/sentry/emails/transactions.html

```diff
@@ -46,5 +46,33 @@
             {% endfor %}
         </tr>
         {% endif %}
+        {% if span_evidence_key_value %}
+        {% for item in span_evidence_key_value %}
+            {% if item.is_multi_value %}
+            <tr>
+                <th>{% trans item.key %}</th>
+                <td>
+                    <code>{{ item.value.0|truncatechars:100 }}</code>
+                </td>
+            </tr>
+            {% for val in item.value|slice:"1:" %}
+            <tr>
+                <th></th>
+                <td>
+                    <code>{{ val|truncatechars:100 }}</code>
+                </td>
+            </tr>
+            {% endfor %}
+
+            {% else %}
+            <tr>
+                <th>{% trans item.key %}</th>
```

Comment:
Ooh, this might not work. `item.key` is a dynamic value, so there's no way to extract a string for a translator to translate 🤔 e.g., on the frontend you can't use `t(` on dynamic values

Reply:
Hmm, i'll look into this, good call 👍

Reply:
Seems to work well
![image](https://user-images.githubusercontent.com/44422760/216679134-50b820a9-dc0f-486f-aeca-edaac667b48d.png)

Reply:
Nice! In that case, maybe `item.name` makes more sense, but not important